src/interface/src/app/plan/create-scenarios/constraints-panel/constraints-panel.component.scss

```diff
@@ -181,16 +181,19 @@
   border: #cccfd8 solid 2px;
 }
 
-::ng-deep .mat-button-toggle-appearance-standard .mat-button-toggle-label-content {
-  line-height: 20px;
-  top: 5px;
-  width: 60px;
+:host {
+  ::ng-deep .mat-button-toggle-appearance-standard .mat-button-toggle-label-content {
+    line-height: 20px;
+    top: 5px;
+    width: 60px;
+  }
+  ::ng-deep .mat-select-value, ::ng-deep .mat-option-text {
```

Comment:
This was leaking styles to all `mat-options-text` - we need to use `ng-deep` with a parent selector to avoid this

Reply:
Thanks for fixing this!